deployment/common/types/types.go

```diff
@@ -33,6 +33,7 @@ type MCMSWithTimelockConfig struct {
 	Bypasser         config.Config
 	Proposer         config.Config
 	TimelockMinDelay *big.Int
+	Label            *string
```

Comment:
the label here is kept as a singular string for now as I'm not sure what the additional use cases would entail, but it can be updated later and shouldn't affect the outcome artifact for existing data should this be changed into a slice or map later